source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRAggregateTarget.mm

```diff
@@ -34,6 +41,10 @@ - (instancetype)initWithActionElement:(ACOBaseActionElement *)actionElement root
 // main entry point to the event handler, override each methods whithin it for custom behaviors
 - (IBAction)send:(UIButton *)sender
 {
+    if (!_doValidation) {
+        [[_view card] setInputs:@[]];
+        [_view.acrActionDelegate didFetchUserResponses:[_view card] action:_actionElement];
+    }
     // dispatch and validate inputs
     ACOInputResults *result = [_view dispatchAndValidateInput:_currentShowcard];
```

Comment:
might be because I'm not very familiar with iOS stuff, but are we sure we still want to do this work if we're not doing validation?

Reply:
@paulcam206 line 45, i'm resetting the inputs that I'm sending to the client, and line 46, I'm sending the message to the clients that their event is triggered.

Reply:
and then we go on to line 49?

Reply:
good point. thanks

Reply:
done